scripts/variantstore/wdl/ImportGenomes.wdl

```diff
@@ -162,20 +172,27 @@ task GetMaxTableId {
 task CreateImportTsvs {
   input {
     File input_vcf
+    File input_vcf_index
     File interval_list
     String output_directory
     File sample_map
+    File? service_account_json
     String? drop_state
     Boolean? drop_state_includes_greater_than = false
 
     # runtime
     Int? preemptible_tries
     File? gatk_override
     String docker
+
+    String? for_testing_only
   }
 
   Int multiplier = if defined(drop_state) then 4 else 10
-  Int disk_size = ceil(size(input_vcf, "GB") * multiplier) + 20
+  #TODO does this affect the memory allocation for the disk?
+  Int disk_size = 1000
```

Comment:
is disk_size not used anywhere? if so could remove this and multiplier variable above? is this what we want, though? (for the disk size not to be scaled with input size - if vcfs are generally a predictable size then it's probably ok to pick a size)

Reply:
the disk_size is used in the runtime section. I can't get the size of the file ahead of time unless I have another task where is auth as the service account and do a gsutil ls -l. @kcibul do you anticipate us running the workflow without dropping a state? If not, we should be able to get rid of the multiplier. My question is whether google will give us a vm with more memory if we ask for this much disk (i'm not sure where this is documented).